whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestOAuthScopeOptionalTestIntegration.java

```diff
@@ -0,0 +1,210 @@
+            "address:   Singel 258\n" +
+            "phone:     +31-1234567890\n" +
+            "e-mail:    noreply@ripe.net\n" +
+            "mnt-by:    OWNER-MNT\n" +
+                "nic-hdl:   PP1-TEST\n" +
+            "remarks:   remark\n" +
+            "source:    TEST\n");
+
+    private static final RpslObject OWNER_MNT = RpslObject.parse("" +
+            "mntner:      OWNER-MNT\n" +
+            "descr:       Owner Maintainer\n" +
+            "admin-c:     TP1-TEST\n" +
+            "upd-to:      noreply@ripe.net\n" +
+            "auth:        MD5-PW $1$d9fKeTr2$Si7YudNf4rUGmR71n/cqk/ #test\n" +
+            "auth:        SSO person@net.net\n" +
+            "mnt-by:      OWNER-MNT\n" +
+            "source:      TEST");
+
+    private static final RpslObject TEST_ROLE = RpslObject.parse(TEST_ROLE_STRING);
+
+    private static final RpslObject TEST_IRT = RpslObject.parse("" +
+            "irt:          irt-test\n" +
+            "address:      RIPE NCC\n" +
+            "e-mail:       noreply@ripe.net\n" +
+            "admin-c:      TP1-TEST\n" +
+            "tech-c:       TP1-TEST\n" +
+            "auth:         MD5-PW $1$d9fKeTr2$Si7YudNf4rUGmR71n/cqk/ #test\n" +
+            "mnt-by:       OWNER-MNT\n" +
+            "source:       TEST\n");
+
+    @Autowired private WhoisObjectMapper whoisObjectMapper;
+    @Autowired
+    private AccessControlListManager accessControlListManager;
+    @Autowired
+    private IpResourceConfiguration ipResourceConfiguration;
+    @Autowired
+    private SSOResourceConfiguration ssoResourceConfiguration;
+    @Autowired
+    private TestPersonalObjectAccounting testPersonalObjectAccounting;
+
+
+    @BeforeAll
+    public static void setupApiProperties() {
+        System.setProperty("apikey.authenticate.enabled","true");
+        System.setProperty("apikey.scope.mandatory","false");
+    }
+
+    @AfterAll
+    public static void restApiProperties() {
+        System.clearProperty("apikey.authenticate.enabled");
+        System.clearProperty("apikey.scope.mandatory");
+        System.clearProperty("apikey.public.key.url");
+    }
+
+    @BeforeEach
+    public void setup() {
+        databaseHelper.addObject("person: Test Person\nnic-hdl: TP1-TEST");
+     //   databaseHelper.addObject("role: Test Role\nnic-hdl: TR1-TEST");
+        databaseHelper.addObject(OWNER_MNT);
+        databaseHelper.updateObject(TEST_PERSON);
+        databaseHelper.addObject(TEST_ROLE);
+        databaseHelper.addObject(TEST_IRT);
+        testDateTimeProvider.setTime(LocalDateTime.parse("2001-02-04T17:00:00"));
+    }
+
+    @AfterEach
+    public void reset() throws Exception {
+        databaseHelper.clearAclTables();
+
+        ipResourceConfiguration.reload();
+        ssoResourceConfiguration.reload();
+        testPersonalObjectAccounting.resetAccounting();
+    }
+
+    @Test
+    public void create_mntner_only_data_parameter_with_apiKey_fails_null_Scope() {
```

Comment:
succeed instead of fails?

Reply:
same for the rest of test here?
I expect that if it is 
` System.setProperty("apikey.scope.mandatory","false");`

The test will succeed if the scope is null